### PR TITLE
Bugfix: houdini hard coded project settings

### PR DESCRIPTION
**Pull request by mustafa-zarkash**
## Changelog Description
I made this PR to solve the issue with hard-coded settings in houdini

## Testing notes:
Change project settings and check if 
1. "Arnold ASS" changes accordingly 
2. ~~Workfile~~ it's automated for validators, check this [comment](https://github.com/ynput/OpenPype/pull/5400#discussion_r1283008864)

### CODE REVIEW

File: openpype/hosts/houdini/plugins/publish/validate_workfile_paths.py [hunk 1]
```diff
@@ -24,6 +24,16 @@ class ValidateWorkfilePaths(
...
 
+    @classmethod
+    def apply_settings(cls, project_settings, system_settings):
+        """Apply project settings to creator"""
+        settings = (
+            project_settings["houdini"]["publish"]["ValidateWorkfilePaths"]
+        )
+
+        cls.node_types = settings.get("node_types", cls.node_types)
+        cls.prohibited_vars = settings.get("prohibited_vars", cls.prohibited_vars)
```
Comment (BigRoy) [inline openpype/hosts/houdini/plugins/publish/validate_workfile_paths.py:35]: Isn't this basically the default behavior already @iLLiCiTiT ? Or do we need to implement this anyway for AYON?

Comment (iLLiCiTiT) [reply openpype/hosts/houdini/plugins/publish/validate_workfile_paths.py:35]: This is indeed defaul behavior.

File: openpype/hosts/houdini/plugins/publish/validate_workfile_paths.py [hunk 2]
```diff
@@ -24,6 +24,16 @@ class ValidateWorkfilePaths(
...
+
```
Comment (iLLiCiTiT) [inline openpype/hosts/houdini/plugins/publish/validate_workfile_paths.py:36]: That is default behavior of automated settings application.
```suggestion
```

Comment (mustafa-zarkash) [reply openpype/hosts/houdini/plugins/publish/validate_workfile_paths.py:36]: So, is this automation happening for vslidators only ? 

---

I found that automation is used in some maya validators. 

However, it's explicitly done in this validator

https://github.com/ynput/OpenPype/blob/develop/openpype/hosts/maya/plugins/publish/validate_maya_units.py#L39-L52

Comment (BigRoy) [reply openpype/hosts/houdini/plugins/publish/validate_workfile_paths.py:36]: The automated behavior is described here](https://openpype.io/docs/dev_publishing#plugin-settings). Not sure if the behavior is also for Creators. :)

Comment (iLLiCiTiT) [reply openpype/hosts/houdini/plugins/publish/validate_workfile_paths.py:36]: It is not for Creators.

Comment (mustafa-zarkash) [reply openpype/hosts/houdini/plugins/publish/validate_workfile_paths.py:36]: Oh, thanks for the link. I will check it.

> Not sure if the behavior is also for Creators.

I kept looking in creators super classes and found there is empty `apply_settings` in creator

**Review by iLLiCiTiT — APPROVED** [review]
LGTM @BigRoy could you test it please?

**Pull request merged**